pycqed/measurement/pulse_sequences/multi_qubit_tek_seq_elts.py

```diff
@@ -844,6 +875,229 @@ def parity_single_round_seq(ancilla_qubit_name, data_qubit_names, CZ_map,
     return seq, np.arange(seq.n_acq_elements())
 
 
+def multi_parity_multi_round_seq(ancilla_qubit_names,
+                                 data_qubit_names,
+                                 parity_map,
+                                 CZ_map,
+                                 prep,
+                                 operation_dict,
+                                 mode='tomo',
+                                 parity_seperation=800e-9,
+                                 rots_basis=('I', 'Y90', 'X90'),
+                                 parity_loops=1,
+                                 cal_points=None,
+                                 prep_params=None,
+                                 upload=True):
+    seq_name = 'Multi_Parity_{}_round_sequence'.format(parity_loops)
+    qb_names = ancilla_qubit_names + data_qubit_names
+
+    # LOL what is this hack?!?!?
+    # 10 points if you can figure out why it is here
+    # 100 points for a general solution where this is not needed
+    dummy_ro_1 = {'pulse_type': 'GaussFilteredCosIQPulse',
+                 'I_channel': 'UHF1_ch1',
+                 'Q_channel': 'UHF1_ch2',
+                 'amplitude': 0.00001,
+                 'pulse_length': 50e-09,
+                 'pulse_delay': 0,
+                 'mod_frequency': 900.0e6,
+                 'phase': 0,
+                 'phi_skew': 0,
+                 'alpha': 1,
+                 'gaussian_filter_sigma': 1e-09,
+                 'nr_sigma': 2,
+                 'phase_lock': True,
+                 'basis_rotation': {},
+                 'operation_type': 'RO'}
+    dummy_ro_2 = {'pulse_type': 'GaussFilteredCosIQPulse',
+                 'I_channel': 'UHF2_ch1',
+                 'Q_channel': 'UHF2_ch2',
+                 'amplitude': 0.00001,
+                 'pulse_length': 50e-09,
+                 'pulse_delay': 0,
+                 'mod_frequency': 900.0e6,
+                 'phase': 0,
+                 'phi_skew': 0,
+                 'alpha': 1,
+                 'gaussian_filter_sigma': 1e-09,
+                 'nr_sigma': 2,
+                 'phase_lock': True,
+                 'basis_rotation': {},
+                 'operation_type': 'RO'}
```

Comment:
This is not really `qudev_master` level code, yes :D
I would guess we can still leave it in for now.